src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (prev, current) =>
+    `${prev}` > `${current}` ? 1
+      : `${prev}` < `${current}` ? -1
+        : 0
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = (prev, current) =>
    `${prev}` > `${current}` 
      ? 1
      : `${prev}` < `${current}` 
        ? -1
        : 0
```


Such code is very hard to read. First of all because of formatting and second - better to use something like `String()` or `.toString`